sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -38,9 +38,10 @@ static char ipv6_str[IPV6_ADDR_MAX_STR_LEN];
 
 static void usage(char **argv)
 {
-    printf("%s [<count>] <ipv6 addr> [<payload_len>] [<delay in ms>] [<stats interval>]\n", argv[0]);
+    printf("%s [<count>] <ipv6 addr[%%<device>]> [<interface>] [<payload_len>] [<delay in ms>] [<stats interval>]\n", argv[0]);
```

Comment:
What's the difference between device any interface here?

Shouldn't the `[%%<device>]` go outside the variable defining `< >` of `<ipv6 addr>`?